src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -526,9 +519,8 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			string modPath = localPath;
 			string path = null;
 			// When JavaScript inserts our path into the html it replaces the three magic html characters with these substitutes.
-			// We need to convert back in order to match our key. Then, reverse the change we made to deal with single quote.
-			// QuoteSubstitute currently contains an &, so we must do that replace last.
-			string tempPath = modPath.Replace("&lt;", "<").Replace("&gt;", ">").Replace("&amp;", "&").Replace(QuoteSubstitute, "'");
+			// We need to convert back in order to match our key. Then, reverse the change we made to deal with quotation marks.
+			string tempPath = UnescapeUrlQuotes(modPath.Replace("&lt;", "<").Replace("&gt;", ">").Replace("&amp;", "&"));
```

Comment:
Like I said, I can't see that the code hurts anything functionally, it just takes time to execute fruitlessly.  And it might conceivably be safer to leave in.